Makefile

```diff
@@ -138,6 +138,7 @@ cache/bats-core-$(BATS_COMMIT).tar.gz:
 
 generate-deploy: pkg/install/generated_templates.gogen.go
 	cd deploy && go run ../pkg/install/generate.go deploy
+	cp ./deploy/flux-helm-release-crd.yaml ./chart/helm-operator/crds/helmrelease.yaml
```

Comment:
Removes the need to manually modify the chart CRD when changes are made,

Reply:
![](https://media.giphy.com/media/ag3PWAeHrCdWV0tlkD/giphy.gif)